### DISCUSSION TITLE
会开发PCL3吗？

### DISCUSSION THREAD

**Discussion by RainbowTaro2022** [Meloong-Git/PCL#1701] [提问] [answered]
我看到PCL2的开发仍在继续，我就是想问一下PCL3会更新ui吗？
现在chatGPT这么火，不会把ChatGPT做进PCL吧

**Comment by Hakuin123** [comment]
ChatGPT和MC启动器毫无关联的好吧😂

**Reply by RainbowTaro2022** [reply]
好吧，那PCL2会一直开发吗？

**Reply by Hakuin123** [reply]
> 好吧，那PCL2会一直开发吗？

这肯定的哇hhh要不然为什么还在更新xwx

**Comment by Chiloven945** [comment]
这个得看情况，之后总会更新的，按照大家的反馈来
~还有就是不要什么事情都提chatGPT实际上它就是一个复制粘贴的程序创作力永远在人身上，或者说是只是一个资本收割的工具（可能有点过激抱歉）~

**Reply by so1ve** [reply]
兄啊，ChatGPT真的能提升生产力（）

**Reply by Chiloven945** [reply]
> 兄啊，ChatGPT真的能提升生产力（）

这的确是事实，但是现在那些人已经吹上天了，我不否认它的生产力但是它也不是那么神圣的东西，还有就是一个启动器要什么生产力……

**Reply by so1ve** [reply]
> > 兄啊，ChatGPT真的能提升生产力（）
> 
> 这的确是事实，但是现在那些人已经吹上天了，我不否认它的生产力但是它也不是那么神圣的东西，还有就是一个启动器要什么生产力……

没有要在PCL里头加的意思（）单纯反驳一下您的观点

**Reply by Chiloven945** [reply]
> > > 兄啊，ChatGPT真的能提升生产力（）
> > 
> > 
> > 这的确是事实，但是现在那些人已经吹上天了，我不否认它的生产力但是它也不是那么神圣的东西，还有就是一个启动器要什么生产力……
> 
> 没有要在PCL里头加的意思（）单纯反驳一下您的观点

嗯……的确有点过激了抱歉，我只是觉得那些人吹得太过头了（

**Reply by so1ve** [reply]
确实（）

**Comment by LTCatt** [comment, marked as answer]
不会（
